robot_state target_position
int32 robot_id
---
bool done